rotkehlchen/constants/assets.py

```diff
@@ -119,6 +119,18 @@
     cryptocompare=None,
 )
 CONSTANT_ASSETS.append(A_KSM)
+A_AVAX = Asset.initialize(
```

Comment:
Ah hey. So this is a not-well documented part of rotki's constant assets. You don't need to add this manually. You just add 

`A_AVAX = Asset('AVAX')`

here:  https://github.com/rotki/rotki/blob/88da5df3d27eb41b7bfb02e6f9d921ac241c8c27/rotkehlchen/constants/assets.py.template#L223

And then you run the script to generate the assets.py


```
python tools/scripts/generate_constant_assets.py
```